src/fragments/lib/geo/android/getting_started/40_init_geo.mdx

```diff
@@ -86,6 +59,34 @@ class MyAmplifyApp : Application() {
 }
 ```
 
+</Block>
+<Block name="RxJava">
+
+```kotlin
+RxAmplify.addPlugin(AWSCognitoAuthPlugin())
+RxAmplify.addPlugin(AWSLocationGeoPlugin())
+RxAmplify.configure(applicationContext)
+```
+
+Your class will look like this:
+
+```kotlin
+class MyAmplifyApp : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        try {
+            RxAmplify.addPlugin(AWSCognitoAuthPlugin())
+            RxAmplify.addPlugin(AWSLocationGeoPlugin())
+            RxAmplify.configure(applicationContext)
+            Log.i("MyAmplifyApp", "Initialized Amplify")
+        } catch (error: AmplifyException) {
+            Log.e("MyAmplifyApp", "Could not initialize Amplify", error)
+        }
+    }
+}
```

Comment:
This code block looks like Kotlin code, but I think it should be Java when using RxJava.